src/test/java/org/springframework/data/elasticsearch/core/index/IndexOperationTransportTests.java

```diff
@@ -0,0 +1,75 @@
+package org.springframework.data.elasticsearch.core.index;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.elasticsearch.annotations.Document;
+import org.springframework.data.elasticsearch.annotations.Mapping;
+import org.springframework.data.elasticsearch.annotations.Setting;
+import org.springframework.data.elasticsearch.core.ElasticsearchOperations;
+import org.springframework.data.elasticsearch.core.IndexOperations;
+import org.springframework.data.elasticsearch.core.mapping.IndexInformation;
+import org.springframework.data.elasticsearch.junit.jupiter.ElasticsearchTemplateConfiguration;
+import org.springframework.data.elasticsearch.junit.jupiter.SpringIntegrationTest;
+import org.springframework.test.context.ContextConfiguration;
+
+import lombok.Data;
+
+/**
+ * @author George Popides
+ */
+
+@ContextConfiguration(classes = { ElasticsearchTemplateConfiguration.class })
+@SpringIntegrationTest
+public class IndexOperationTransportTests {
```

Comment:
It's not necessary to implement anything in this class. Just like I commented last time, all that's needed is a derived class with a different `@ContextConfiguration` annotation. When the tests are run, this derived class will set up the `ElasticsearchOperations` using the transport client and run the tests with this client. 
The `IndexOperationsTests` class just uses the methods from `ElasticsearchOperations` not caring which implementation it gets.

Reply:
I tried to do it in this way, but the the tests when were run all together were failing with the following exception:
``` Elasticsearch exception [type=resource_already_exists_exception, ...``` for ```test-index-rest-information-list``` index.

I removed one of the entities and left just one with index name: ```test-index-information-list``` but the test still failed.
I have tried to remove ```indexOps.create()``` from one of the methods, but it didn't work either.

If the test is run alone, it passes.  So that's why i didn't extend the ```IndexOperationTests``` class in previous commit.

Reply:
forgot that. You need a method
```java
    @BeforeEach
    void setUp() {
        operations.indexOps(EntityWithSettingsAndMappingsRest.class).delete();
    }
```
in the base class that deletes the index on test start when it exists.